scripts/deployment.ts

```diff
@@ -1,19 +1,17 @@
 import { ethers } from 'hardhat'
 import '@nomiclabs/hardhat-ethers'
 import { deployAllContracts } from './deploymentUtils'
-import { maxDataSize } from './config'
+import { maxDataSize, ESPRESSO_LIGHT_CLIENT_ADDRESS } from './config'
 
 async function main() {
   const [signer] = await ethers.getSigners()
-
   try {
-    // Deploying all contracts
     const contracts = await deployAllContracts(
       signer,
       ethers.BigNumber.from(maxDataSize),
-      true
+      true,
+      ESPRESSO_LIGHT_CLIENT_ADDRESS
```

Comment:
I think we can add this to `.env` so we don't have to make any change to `config.ts`. Also let's log a warning if it's not set: (e. g. something like `env var ESPRESSO_LIGHT_CLIENT_ADDRESS not set, it needs to be set to deploy the RollupCreator for the espresso integration`)

Reply:
Sounds good! Will do